app/code/community/Bolt/Boltpay/Model/Admin/ExtraConfig.php

```diff
@@ -184,15 +216,15 @@ function(hints) {
             : $defaultFunction;
     }
 
+
     /**
      * @param $rawConfigValue
      * @param array $additionalParams
      * @return string
      */
     public function filterDatadogKeySeverity($rawConfigValue, $additionalParams = array())
     {
-        $allExtraConfigs = (array)json_decode($this->normalizeJSON(Mage::getStoreConfig('payment/boltpay/extra_options')), true);
-        return (array_key_exists("datadogKeySeverity", $allExtraConfigs)) ? $rawConfigValue : Bolt_Boltpay_Helper_DataDogTrait::$defaultSeverityConfig;
+        return strlen(trim($rawConfigValue)) ? trim($rawConfigValue) : Bolt_Boltpay_Helper_DataDogTrait::$defaultSeverityConfig;
```

Comment:
I don't think the change above will work when we want to disable sending logs to DataDog as default severity will always be used.

Reply:
@johnmccombs1 
As I understand from what you have written is that our method for disabling datadog is setting the `datadogKeySeverity` to an empty string, but if we want the default value, we omit defining that same key.

This is probably not as straight forward as having an enable setting or disabling by not providing a value for ``datadogKey`, but, we can discuss that in a separate PR.  For the sake of moving this PR forward, I'm reverting that above code.